docs/guide/ops/cli/cli-ref-guide.md

```diff
@@ -0,0 +1,310 @@
+  Deploy an application based on the supplied YAML file or read from STDIN when `-` is given instead of a file name.
+
+- `br application`  
+  List the running applications.
+
+- `br application <Name|AppID>`  
+  Show the detail for an application.
+
+- `br <app-scope> config`  
+  Show the configuration details for an application.
+
+- `br <app-scope> config <ConfigID>`  
+  Show the value for a configuration item.
+
+- `br <app-scope> spec`  
+  Show the YAML specification used to create the application.
+
+- `br <app-scope> rename <Name>`  
+  Rename the application to <Name>.
+
+- `br <app-scope> stop`  
+  Stop an application.  See below for further information on the `stop` effector.
+
+- `br <app-scope> start`  
+  Start an application.  See below for further information on the `start` effector.
+
+- `br <app-scope> restart`  
+  Restart an application.  See below for further information on the `restart` effector.
+
+- `br <app-scope> delete`  
+  Delete an application from Brooklyn.  
+  **NOTE:** Use this command with care.  Even if the application / entities are still running, Brooklyn will drop all knowledge of them and they will be left running in an 'orphaned' state.
+
+### Entities
+
+- `br <app-scope> entity`    
+  List the child entities for an application.
+
+- `br <entity-scope> entity`  
+  List the child entities for an entity.
+
+- `br <app-scope> entity <Name|EntityID>`  
+  Show the detail of an entity.
+
+- `br <app-scope> entity -c <Name|EntityID>`  
+  List the child entities for an entity.
+
+- `br <entity-scope> config`  
+  Show the configuration details for an entity.
+
+- `br <entity-scope> config <ConfigID>`  
+  Show the value for a configuration item.
+
+- `br <config-scope> set <ConfigValue>`  
+  Set the value of a configuration item.  
+
+- `br <entity-scope> spec`  
+  Show the YAML specification used to create the entity.
+
+- `br <entity-scope> rename <Name>`  
+  Rename the entity to <Name>.
+
+- `br <entity-scope> stop`  
+  Stop an entity.  See below for further information on the `stop` effector.
+
+- `br <entity-scope> start`  
+  Start an entity.  See below for further information on the `start` effector.
+
+- `br <entity-scope> restart`  
+  Restart an entity.  See below for further information on the `restart` effector.
+
+### Sensors
+
+- `br <app-scope> sensor`  
+  List the sensors and values for an application.
+
+- `br <app-scope> sensor <SensorID>`  
+  Show the value for a sensor.
+
+- `br <entity-scope> sensor`  
+  List the sensors and values for an entity.
+
+- `br <entity-scope> sensor <SensorID>`  
+  Show the value for a sensor.
+
+### Effectors
+
+- `br <app-scope> effector`  
+  List the effectors for an application.
+
+- `br <app-scope> effector <EffectorID>`  
+  Show the detail for an application effector.
+
+- `br <app-scope> effector <EffectorID> invoke`  
+  Invoke the effector without any parameters.
+
+- `br <app-scope> effector <EffectorID> invoke [<param>=<value> ...]`  
```

Comment:
Is there any way of passing a file to a parameter rather than simple value, for example when you are submitting multi-line yaml to an effector.

(cc @m4rkmckenna)